universal-login-react/src/ui/commons/EnsName.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+interface EnsNameProps {
+  value: string;
+}
+
+export const EnsName = ({value}: EnsNameProps) => {
+  const [name, ...domain] = value.split('.');
```

Comment:
use `parseDomain`

Reply:
parseDomain returns just domain. We also need a name here

Reply:
@JustynaBroniszewska  Done